src/Bridges/Nette/TracyExtension.php

```diff
@@ -19,6 +19,8 @@ class TracyExtension extends Nette\DI\CompilerExtension
 {
 	public $defaults = array(
 		'email' => NULL,
+		'logDirectory' => NULL,
+		'logSeverity' => 0,
```

Comment:
What if we change `if ($value !== NULL)` to `if ($value !== $this->defaults[$key])`?